arcor2_AREditor/ProjectSettings/ProjectSettings.asset

```diff
@@ -838,7 +838,7 @@ PlayerSettings:
   webGLMemoryGeometricGrowthCap: 96
   webGLPowerPreference: 2
   scriptingDefineSymbols:
-    Android: UNITY_POST_PROCESSING_STACK_V2;ARFOUNDATION_PRESENT;AR_ON
+    Android: UNITY_POST_PROCESSING_STACK_V2;ARFOUNDATION_PRESENT
```

Comment:
Please keep the AR_ON symbol there by default...

Reply:
Done